src/commands/rolls/combat.ts

```diff
@@ -120,12 +120,14 @@ class CombatRollCommand extends Command {
         name: "main",
         content: new MessageEmbed({
           ...getAuthorData(this.client),
-          title: success ? "Success!" : "Failure!",
-          color: success ? successColor : failureColor,
+	  title: damage > 0 ? (damage >= 5 ? "CRITICAL!": "Hit!") : "Miss!",
```

Comment:
So my problem here is that 5 damage does not necessarily deal a critical hit. Critical hits have to be determined after damage reduction is taken into account, and **BIG** creatures have a critical threshold of 7: 

```
BIG: The [creature] is bigger than most
characters, towering over them. The creature
receives an additional +1 health point per
Level, but its Defense decreases by 1, to a
minimum of 1. Further, it only suffers a
Critical Hit if an attack inflicts 7+ damage
(after damage resistance) in a single hit,
rather than the normal 5+.
```

I do like the more descriptive and flavorful output, but I am not sure this is the exact approach here. Maybe we remove the logic around critical hits, keep the new output you wrote for standard hits or misses?

- [ ] Remove critical hit logic (cannot know if it is actually a critical)

Reply:
Hmm, that is valid. I'll cut it.